src/ServiceManager/Travely.ServiceManager.Service/Mappers/ActivityProfile.cs

```diff
@@ -0,0 +1,9 @@
+﻿using AutoMapper;
+
+namespace Travely.ServiceManager.Service.Mappers
+{
+    public class ActivityProfile : Profile
```

Comment:
Not used. Why this has been added?

Reply:
I'll use this when we have db layer models and unit of works, mentioned this also in pr's descriptions